ansible-shell

```diff
@@ -339,23 +422,15 @@ class AnsibleShell(cmd.Cmd):
         mline = line.partition(' ')[2]
         offs = len(mline) - len(text)
 
-        if self.options.cwd == '*':
-            completions = self.hosts + self.groups
+        if self.cli.options.cwd == '*':
+            completions = self.cli.hosts + self.groups
         else:
-            completions = self.ansible.inventory.list_hosts(self.options.cwd)
-
-        return [s[offs:] for s in completions if s.startswith(mline)]
-
-    def completedefault(self, text, line, begidx, endidx):
```

Comment:
this shouldn't be removed because this provides the tab completion for the module parameters